src/dialects/sqlite/query-generator.js

```diff
@@ -3,10 +3,10 @@
 const Utils = require('../../utils');
 const { Transaction } = require('../../transaction');
 const _ = require('lodash');
-const MySqlQueryGenerator = require('../mysql/query-generator');
-const AbstractQueryGenerator = require('../abstract/query-generator');
+const { MySqlQueryGenerator } = require('../mysql/query-generator');
```

Comment:
Interesting to see it here as well, that these dialects depend on eachother.

Reply:
This one is a lot more surprising to me